agent/asm/asm.go

```diff
@@ -32,6 +34,32 @@ type AuthDataValue struct {
 	Password *string
 }
 
+func resourceInitializationErrMsg(secretID string) string {
+	return fmt.Sprintf(
+		`ResourceInitializationError: The task can't retrieve the secret with ARN %sfrom AWS Secrets Manager. Check that the secret ARN is correct`,
+		secretID)
+}
+
+// Augment error message with extra details for most common exceptions:
+func augmentErrMsg(secretID string, err error) string {
+	if secretID != "" {
+		secretID = "'" + secretID + "' "
+	} else {
+		logger.Warn("augmentErrMsg: SecretID is empty (which is unexpected)")
+	}
+	if aerr, ok := err.(awserr.Error); ok {
+		switch aerr.Code() {
+		case secretsmanager.ErrCodeResourceNotFoundException:
+			fmt.Println(aerr.Message())
```

Comment:
Probably want to delete this `fmt.Println` call

Reply:
Thanks!